dts/bindings/mspi/espressif,esp32-mspi.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2025, Pavel Maloletkov
+# SPDX-License-Identifier: Apache-2.0
+
+description: Espressif ESP32 MSPI controller
+
+compatible: "espressif,esp32-mspi"
+
+include: [mspi-controller.yaml, pinctrl-device.yaml, reset-device.yaml]
+
+properties:
+  reg:
+    required: true
+    description: Base address and size of the SPI peripheral registers
+
+  peripheral-id:
+    type: int
+    required: true
+    description: |
+      ESP32 SPI peripheral ID (0 or 1)
+      - 0: SPI1 (HSPI)
+      - 1: SPI2 (VSPI)
+      Note: SPI0 is reserved for flash cache and not available for general use
+
+  clocks:
+    required: true
+    description: Clock controller phandle and clock specifier
+
+  clock-frequency:
+    type: int
+    default: 80000000
+    description: |
+      Maximum supported frequency in Hz. Default is 80MHz.
+      This should not exceed the capabilities of the selected clock source.
+
+  interrupts:
+    description: Interrupt specifier for the SPI peripheral
+
+  dma-enabled:
+    type: boolean
+    description: Enable SPI DMA support
+
+  dma-clk:
+    type: int
+    description: DMA clock source
+
+  dma-host:
+    type: int
+    description: DMA Host - 0 -> SPI2, 1 -> SPI3
+
+  line-idle-low:
+    type: boolean
+    description: |
+      Default data lines GPIO level when idle. Defaults to high by default.
+
+  max-dma-buf-size:
+    type: int
+    default: 4092
```

Comment:
Bindings property have the purpose to hint the system about the presence of a hardware resource if it needs to be configurable let's say by software it should be an option for the driver's kconfig

Reply:
Done